### PR TITLE
to adjust the width of transform dialog. issue #1332

### PR DESCRIPTION
**Pull request by jackyq2015**
change to 80% from 100%

### CODE REVIEW

File: main/webapp/modules/core/scripts/views/data-table/text-transform-dialog.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="dialog-frame" >
+<div class="dialog-frame" style="width: 80%;" >
```
Comment (wetneb) [inline main/webapp/modules/core/scripts/views/data-table/text-transform-dialog.html]: Wouldn't it be cleaner to store that in CSS? It's generally recommended to avoid inline styling.

Comment (ostephens) [reply main/webapp/modules/core/scripts/views/data-table/text-transform-dialog.html]: The approach @jackyq2015 has taken here is consistent with all the other 'dialog-frame' divs in the app at the moment. IMO ideally all these should be done instead by adding a class to the div and styling via css. BUT, I'd suggest consistency is better than doing this one instance 'correctly' - then we can tidy them all together (I'd be happy to take this on once this PR is merged)

Comment (jackyq2015) [reply main/webapp/modules/core/scripts/views/data-table/text-transform-dialog.html]: agree

File: refine [hunk 2]
```diff
@@ -942,7 +942,7 @@ elif [ "$OS" = "linux" ] ; then
 fi
 
 echo You have "$freeRam"M of free memory. 
-echo Your current configuration will allow to use $REFINE_MEMORY of memory.
+echo Your current configuration is set to use $REFINE_MEMORY of memory.
```
Comment (wetneb) [inline refine]: This PR contains two unrelated changes… It's generally better to use separate branches and PRs for that.

File: main/webapp/modules/core/scripts/views/data-table/text-transform-dialog.html (hunk 1 above)
Comment (thadguidry) [reply main/webapp/modules/core/scripts/views/data-table/text-transform-dialog.html]: Team, we know its best practice, but let's not focus too much on the frontend incidentals.  Our plan is for a UI overhaul with Google's help.  Honestly, we just need to work on the backend and just "fast quick repair" on the frontend to make things "work good enough for now", knowing that our UI re-work can happen with Google's assistance in a bit.  I already talked to Jacky about this during my hang out with him.  Simon@Google is working on getting a designer or two for helping us out.

Speaking of hangouts, we need to come to some regularly scheduled hangouts for the 4 of us.  Oh @ostephens if you didn't know, we kinda voted you as a core contributor in the past day :) and I'll send out an official email shortly, announcing.

File: refine (hunk 2 above)
Comment (thadguidry) [reply refine]: @wetneb We know.  Jacky is just fixing some incidentals that I bring up in our hangouts.  There just minor typo fixes. I know, I know, it makes the changes look dirty. I know.  But for typos, I am OK with dirty changes of typos within feature pulls.  I know your probably not, but I and Jacky are OK with them.  We're moving fast, because we have an agenda to get this release done, moved out...so that we can actually begin to plan better and focus on "the 1st quarter fun".
